config/routes.rb

```diff
@@ -43,5 +43,6 @@
     jsonapi_resources :enablings do; end
     jsonapi_resources :comments, except: :show do; end
     jsonapi_resources :external_sources do; end
+    jsonapi_resources :sessions do; end
```

Comment:
Please add `, only: [:create]` to this line, to make sure this is the only endpoint created.